rllib/env/env_runner.py

```diff
@@ -69,6 +75,22 @@ def __init__(self, *, config: "AlgorithmConfig", **kwargs):
         ):
             tf1.enable_eager_execution()
 
+        # Determine actual seed for this particular worker based on worker index AND
+        # whether it's an eval worker.
+        self._seed: Optional[int] = None
+        if self.config.seed is not None:
+            self._seed = int(
+                self.config.seed
+                + (self.worker_index or 0)
```

Comment:
Dumb question: The worker index stays the same when a work crashes and is replaced, correct?